runtime/defaults/src/main/resources/application.properties

```diff
@@ -122,9 +122,27 @@ polaris.features."SUPPORTED_CATALOG_CONNECTION_TYPES"=["ICEBERG_REST"]
 # polaris.persistence.type=eclipse-link
 # polaris.persistence.type=in-memory-atomic
 polaris.persistence.type=in-memory
-# polaris.persistence.type=relational-jdbc
-
+#polaris.persistence.type=relational-jdbc
 polaris.secrets-manager.type=in-memory
+#disable localstack
+#if set to true it will try to start localstack at build and run time for the local environment
+quarkus.devservices.enabled=false
+
+
+#AWS IAM postgres example
```

Comment:
wondering linking this additionally https://docs.quarkiverse.io/quarkus-amazon-services/dev/amazon-rds.html#_configuration_reference would be helpful too